### DISCUSSION TITLE
[SafelinkScript] /js/full-page-script.js

### DISCUSSION THREAD

**Discussion by realodix** [realodix/AdBlockID#731] [Ideas]
> ini kemungkinan blocking script filternya jg sama gan sumber https://psarips.eu/
> <details>
>
> https://earnload.com/full/?api=9e215342d130cd952872648af56e6247ab403c10&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv
> 
> http://shrinkme.io/full/?api=6a4e9296859080eb5c9c8d797358d9069754edbc&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv
> 
> https://links4earn.com/full/?api=23f0ca5680d15a3a33311f2fd0220324d004a38c&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv
> 
> https://shorten.sh/full/?api=4cad1dae3630a2d2d580570ec5c75f8be5011a6f&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv
> 
> https://cutpaid.com/full/?api=5e028c40b1b4d3e70c504929e780acc4a701fd98&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv
> 
> https://ilinkshort.com/full/?api=507454267fef0dbc50f6dd90769cb9c468458a5d&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv
> 
> https://urle.io/full/?api=726f220bb154f22e398a30e130fc7531b019ee14&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv
> 
> https://gpmojo.com/full/?api=180ffb45ee1d5bed5c471a6e1c11adf29b994376&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv
> 
> https://boomx5.com/full/?api=c8212e5bd65ce2675d9a5913462be404&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv
> 
> https://gplinks.in/full/?api=a34e4c05ef8afa7330c7dc7da6cf9fd84623e91b&url=aHR0cHM6Ly9nZXQtdG8ubGluay90aGUtd2l0Y2hlci1zMDFlMDgtbXVjaC1tb3JlLTcyMHAtMTBiaXQtd2VicmlwLTJjaC14MjY1LWhldmMtcHNhNjc0NTQv</details>

_Originally posted by @WaferNabati in https://github.com/realodix/AdBlockID/issues/117#issuecomment-568465899_

Terima kasih sudah membantu menemukan lebih banyak safelink script.

Link di atas, halaman `/js/full-page-script.js` terlihat mirip banget.
https://earnload.com/js/full-page-script.js
https://shrinkme.io/js/full-page-script.js
http://links4earn.com/js/full-page-script.js
http://shorten.sh/js/full-page-script.js
https://cutpaid.com/js/full-page-script.js
http://ilinkshort.com/js/full-page-script.js
https://urle.io/js/full-page-script.js
https://gpmojo.com/js/full-page-script.js
https://gplinks.in/js/full-page-script.js

Hanya boomx5.com Saja yang terlihat berbeda
http://boomx5.com/js/full-page-script.js

---

Saya sedang memikirkan solusi yang bagus, terdapat 2 solusi untuk saat ini
1. Memasukkan satu per satu
Cukup aman, tapi filter jadi terlihat gendut.

2. Cukup dengan `/js/full-page-script.js$script,third-party`
Sangat efisien, tapi belum tentu efektif. Walaupun pakai embel2 `$third-party`, tapi `full-page-script.js` telihat umum. Butuh pembelajaran lebih lanjut, takutnya bisa merusak banyak situs.

**Comment by realodix** [comment]
Masih dari https://psarips.eu/

https://oko.sh/1k6cLK
https://oko.sh/js/full-page-script.js

https://ckk.ai/hLR5c
https://ckk.ai/js/full-page-script.js

https://fc.lc/QYc2
https://fc.lc/js/full-page-script.js

https://tmearn.com/zBHvb
https://tmearn.com/js/full-page-script.js

https://spiin.xyz/vti77
https://spiin.xyz/js/full-page-script.js

https://links4earn.com/8igkLPg
https://links4earn.com/js/full-page-script.js

**Comment by Recehan-Slayer** [comment]
http://shrtwin.com/js/full-page-script.js
http://ventax.net/js/full-page-script.js
http://voxc.org/js/full-page-script.js
https://v6download.com/js/full-page-script.js

**Comment by Recehan-Slayer** [comment]
bnyak bgt gan klo 1 per satu pkoknya yg pke template adlinkfly hmpir smaa semua

**Comment by realodix** [comment]
> bnyak bgt gan klo 1 per satu pkoknya yg pke template adlinkfly hmpir smaa semua

Umum banget nama file `full-page-script.js`, kan bisa aja ada situs di luar sana yang panggil `full-page-script.js` dari pihak ke tiga yang isinya untuk full screen. Tapi memang cari di Google dengan keyword `full-page-script` isinya safelink semua di halaman pertama.

**Comment by Recehan-Slayer** [comment]
ya coba aja pke opsi ke 2 dlo ntar klo ada orang laporan ada salah blockir atau apa baru deh cari solusi yg lain

**Comment by realodix** [comment]
> bnyak bgt gan klo 1 per satu pkoknya yg pke template adlinkfly hmpir smaa semua

Iya sih bener juga. Saya coba deh opsi ke 2 dengan tetap membiarkan issue ini terbuka

**Comment by Recehan-Slayer** [comment]
@realodix mntab bro bnyk bgt web" yg 3dp scriptnya terblock contohnya ini shortlink indo
`https://layarsoft.com/idm-terbaru-fix-full.html`
`http://www.ovagames.com/pro-cycling-manager-2019-skidrow.html`
`https://lk21movie.online/the-kid-who-would-be-king-2019/`
`https://moviesrex.in/download-the-lighthouse-movie-2019-bluray-1080p/`

**Comment by realodix** [comment]
> mntab bro bnyk bgt web" yg 3dp scriptnya terblock contohnya ini shortlink indo
> `https://layarsoft.com/idm-terbaru-fix-full.html`

Karena ente juga Bro yg sudah menyarankan dan membantu. Thanks Bro

**Comment by realodix** [comment]
https://www.wts.pw/js/full-page-script.js
https://www.adtival.network/js/full-page-script.js

Ditemukan di https://doramaindo.id/karera-wo-mireba-wakaru-koto-subtitle-indonesia.html

**Comment by realodix** [comment]
Saat ini di https://github.com/realodix/AdBlockID/issues/379, memiliki bentuk yang sedikit melenceng. Tidak menggunakan pola `/js/` di depannya.
https://rawcdn.githack.com/KecoaTerbank/safelink/9530efb06c5124104e2bdedb012a90ed4e06782b/full-page-script.js

Solusi pomblokiran langsung diblokir 1 repository

**Comment by realodix** [comment]
Ditemukan pada https://github.com/realodix/AdBlockID/issues/411
- https://linkspy.cc/js/fullPageScript.min.js
